src/segment-parser.js

```diff
@@ -214,6 +214,16 @@
         // adaptation_field_control, whether this header is followed by an
         // adaptation field, a payload, or both
         afflag = (data[offset + 3] & 0x30 ) >>> 4,
+        adaptation_field_length,
+        afftemp,
+        discontinuity_indicator,
+        random_access_indicator,
```

Comment:
I know this code is very inconsistent about variable naming but we should stick with camel case.